apps/chat/src/components/Chatbar/Conversation.tsx

```diff
@@ -679,6 +678,7 @@ export const ConversationComponent = ({ item: conversation, level }: Props) => {
       {isPublishing && (
         <PublishModal
           entity={conversation}
+          entities={[conversation]}
```

Comment:
Why do we need it if we have `entity={conversation}`?

Reply:
this was done to throw child entities if we call this window for folders.

Reply:
In this case let's remove the property `entity={conversation}` and put main resource as first in `entities`

Reply:
This will make it very difficult to understand the code, since entities can contain both a folder and a chat or prompt